src/engines/misc/sends/sends.hpp

```diff
@@ -37,6 +49,9 @@ namespace otto::engines::sends {
 
     Props props;
 
+  private:
+    bool is_recursive = false;
```

Comment:
for being a necessary evil, this `is_recursive` isn't that bad. But it needs a doc comment! mostly to describe that its just used in the on_change signals to stop them from recusing